dynamic_programming/longest_increasing_subsequence.py

```diff
@@ -4,50 +4,50 @@
 This is a pure Python implementation of Dynamic Programming solution to the longest increasing subsequence of a given sequence.
 
 The problem is  :
-Given an ARRAY, to find the longest and increasing sub ARRAY in that given ARRAY and return it.
+Given an array, to find the longest and increasing sub ARRAY in that given array and return it.
 Example: [10, 22, 9, 33, 21, 50, 41, 60, 80] as input will return [10, 22, 33, 41, 60, 80] as output
 """
 from typing import List
 
 
-def longestSub(ARRAY: List[int]) -> List[int]:  # This function is recursive
+def longest_subsequence(Array: List[int]) -> List[int]:  # This function is recursive
```

Comment:
Array --> array please.  Please see CONTRIBUTING.md for info on variable naming in Python.

Reply:
Woo, my bad. Followed strictly the PEP8.